### ISSUE TITLE
Configure cfn-hup to update machines on metadata changes

### ISSUE THREAD

**Issue by villasv** [villasv/turbine#72]
One of the regressions after migrating to `systemd`.

**Comment by villasv** [comment]
For reference, this is how it used to be:

```
        formation:
          files:
            /etc/cfn/cfn-hup.conf:
              content: !Sub |
                [main]
                stack=${AWS::StackId}
                region=${AWS::Region}
                interval=1
              mode: '000400'
              owner: root
              group: root
            /etc/cfn/hooks.d/cfn-auto-reloader.conf:
              content: !Sub |
                [cfn-auto-reloader-hook]
                triggers=post.update
                path=Resources.Scheduler.Metadata.AWS::CloudFormation::Init
                action=/opt/aws/bin/cfn-init --resource Meta --stack ${AWS::StackName} --region ${AWS::Region}
                runas=root
          services:
            sysvinit:
              cfn-hup:
                enabled: true
                ensureRunning: true
                files:
                  - /etc/cfn/cfn-hup.conf
                  - /etc/cfn/hooks.d/cfn-auto-reloader.conf
```

**Comment by villasv** [comment]
Solved after 93a20a1, since restarting Airflow was required after changing the environment variables.
